### ISSUE TITLE
Frequenchy jumps around in WSJT-X

### ISSUE THREAD

**Issue by Uncajac** [km4ack/73Linux#125]
when xmitting in WSJT-X the frequency jumps all over the place. The radio is usually not affected. FLDigi usually is not affected but occasionally will die. Receiving is not affected. Radio - Icom IC-706MKIIG, Antenna - 80M EFHW.

**Comment by km4ack** [comment]
I don't think that is a software issue. Educated guess says RFI is getting into your system.
